### PR TITLE
Add rotation angle to orientation attribute on page level

### PR DESCRIPTION
**Pull request by wrznr**
With PAGE XML 2019, we can add skew angle on page level
appropriately!

### CODE REVIEW

**Comment by wrznr** [comment]
@n00blet FYI

Serves as best practice how to represent/store deskewing results in PAGE.

**Comment by n00blet** [comment]
@wrznr Thanks a lot. I will look into it.

File: ocrd_cis/ocropy/binarize.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def process_page(self, page, page_image, page_xywh, page_id, file_id):
         # annotate angle in PAGE (to allow consumers of the AlternativeImage
         # to do consistent coordinate transforms, and non-consumers
         # to redo the rotation themselves):
-        #page.set_orientation(-angle) # FIXME does not exist on page level yet!
+        page.set_orientation(-angle)
```
Comment (wrznr) [inline ocrd_cis/ocropy/binarize.py:164]: Not sure about this one. Why would you want to undo deskewing while binarising?

**Comment by wrznr** [comment]
@bertsky Can I close this one?

**Comment by bertsky** [comment]
> Can I close this one?

Yes, please do. I'm sorry – I did not notice your PR at the time.

**Pull request closed without merging**